lib/jormungandr/test/data/jormungandr/node.config

```diff
@@ -0,0 +1,10 @@
+storage: "/tmp/cardano/storage"
```

Comment:
Have to make sure this exists when doing `beforeAll`

Reply:
I'd suggest something like `/tmp/cardano-wallet-jormungandr` to be more consistent with the bridge. Also, yes, the directory should probably be created. Not sure how Jormungandr handles it.